Deploy/Build.ps1

```diff
@@ -48,14 +55,6 @@ Else
 
 # Use Git to figure out changes
 $uwpPath = $RootPath + '\Deploy\uwp_jcadmu.ps1'
-$changes = git diff origin/master... -- $uwpPath
-
-if (-not [System.String]::IsNullOrEmpty($changes))
-{
-    Invoke-ps2exe -inputFile ($uwpPath) -outputFile ($RootPath + '\jumpcloud-ADMU\exe\uwp_jcadmu.exe') -title 'JumpCloud ADMU UWP Fix' -product 'JumpCloud ADMU' -description 'JumpCloud AD Migration Utility UWP Fix Executable' -copyright '(c) 2021' -company 'JumpCloud' -iconfile ($RootPath + '\Deploy\admu.ico')
-    Write-Host "upw_jcadmu.exe was generated successfully"
-}
-else
-{
-    Write-Host "No changes to uwp_jcadmu.ps1 file"
-}
\ No newline at end of file
+# Always generate a new UWP EXE
+Invoke-ps2exe -inputFile ($uwpPath) -outputFile ($RootPath + '\jumpcloud-ADMU\exe\uwp_jcadmu.exe') -title 'JumpCloud ADMU UWP Fix' -product 'JumpCloud ADMU' -description 'JumpCloud AD Migration Utility UWP Fix Executable' -copyright "(c) $year" -version $Version.Matches.Value -company 'JumpCloud' -iconfile ($RootPath + '\Deploy\admu.ico')
```

Comment:
Here I'm always generating the uwp exe instead of checking the git diff. 

1. we'll always publish a new version 
2. we'll update the copyright data dynamically with the changes on line 59